example/README.md

```diff
@@ -768,12 +769,77 @@ After calling `startHLSStreaming` we will get an `onSuccess` callback if the met
 100ms Flutter SDK provides support for creating a Picture in Picture mode experience for video calls.
 
 PIP Mode lets the user watch the room video in a small window pinned to a corner of the screen while navigating between apps or browsing content on the main screen.
-Currently, this functionality is only available on Android.
 
 To know more about how to implement PIP mode. Please check: [PIP Mode](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
 
+- PIP in Android
+
 https://user-images.githubusercontent.com/93931528/205587304-772a5dd6-ed64-4d9e-8bb5-4fc3eed83bea.mp4
 
+- PIP in iOS
+
+https://github.com/100mslive/100ms-flutter/assets/93931528/af331f42-fe33-4345-bf6b-e623cc2d29e8
+
+### 17. Spotlight
```

Comment:
@Decoder07 Nice 💯
Let's add this to the Session Store page on Public Docs as well